### PR TITLE
[DOC-XXX] [fix] - Settings module moved to coffeescript - #644

### PR DESCRIPTION
**Pull request by rsvato**

### CODE REVIEW

File: ui/src/main/resources/app/modules/settings/configs.coffee [hunk 1]
```diff
@@ -0,0 +1,112 @@
...
+    toggleShowKey: (event) ->
+      $link = $(event.currentTarget)
+      $span = $link.children("span")
+      $span.toggleClass "ui-icon-carat-1-n ui-icon-carat-1-s"
+      $span.attr "title", (if $span.hasClass("ui-icon-carat-1-s") then "Show Key" else "Hide Key")
+      $element = $($link.attr("rel"))
+      $element.toggle()
+
+    showConfirmationDialog: ->
+      self = this
```
Comment (ddurnev) [inline ui/src/main/resources/app/modules/settings/configs.coffee:92]: Is it necessary here?

File: ui/src/main/resources/app/modules/settings/groups.coffee [hunk 2]
```diff
@@ -0,0 +1,177 @@
...
+          @render()
+      else
+        $.when(@users.fetch()).done _.bind(@render, this)
+
+    render: ->
+      self = this
+      $.when(genesis.fetchTemplate(@template), backend.AuthorityManager.roles()).done (tmpl, rolesArray) =>
+        usersInGroupMap = genesis.utils.toBoolean(@groupUsers.pluck("username"))
+        groupRolesMap = genesis.utils.toBoolean(_(@groupRolesArray))
+        self.$el.html tmpl(
```
Comment (ddurnev) [inline ui/src/main/resources/app/modules/settings/groups.coffee]: @$el?

File: ui/src/main/resources/app/modules/settings/plugins.coffee [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+
+  class PluginSettings extends Backbone.View
+    template: "app/templates/settings/plugin_settings.html"
+    events:
+      "click a.back": "backToPlugins"
+      "click a.save": "savePluginSettings"
+
+    initialize: (options) ->
+      @plugin = @model
+      self = this
```
Comment (ddurnev) [inline ui/src/main/resources/app/modules/settings/plugins.coffee]: self is not used?

File: ui/src/main/resources/app/modules/settings/configs.coffee (hunk 1 above)
Comment (rsvato) [inline ui/src/main/resources/app/modules/settings/configs.coffee:92]: Yup. There is dialog function, and, as I remember from agents.coffee, there is some problems with using fat arrow in it.

**Pull request merged**